src/pages/_app.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import ReactGa from 'react-ga';
+import PropTypes from 'prop-types';
+import Head from 'next/head';
+import { I18nextProvider } from 'react-i18next';
+import { ToastContainer } from 'react-toastify';
+import * as Sentry from '@sentry/react';
+import { MuiThemeProvider } from '@material-ui/core/styles';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { APP_VERSION, SENTRY_DSN, GOOGLE_ANALYTICS_ID } from '../config/env';
+import i18nConfig from '../config/i18n';
+import WHITELISTED_ERRORS from '../config/errors';
+import { APP_NAME } from '../config/constants';
+import theme from '../config/theme';
+
+// set up sentry
+Sentry.init({
+  dsn: SENTRY_DSN,
+  ...WHITELISTED_ERRORS,
+  beforeSend(event) {
+    // check if it is an exception, and if so, show the report dialog
+    if (event.exception) {
+      Sentry.showReportDialog({ eventId: event.event_id });
+    }
+    return event;
+  },
+  release: `${APP_NAME} ${APP_VERSION}`,
+});
+
+// set up google analytics
+if (typeof window !== 'undefined') {
+  ReactGa.initialize(GOOGLE_ANALYTICS_ID);
+  ReactGa.pageview(window.location.href);
+}
+
+export default function MyApp(props) {
```

Comment:
Can we rename `MyApp` to something like `GraaspExplorer` or `GraaspExplorerApp`?